### ISSUE TITLE
Informations about Service should become configurable

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#45]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/45) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

We should introduce to set this text used in Service in mss_wms_settings and fill into the template

```
#!text

<Service>
        <Name>OGC:WMS</Name>
        <Title>Mission Support System Web Map Service</Title>
        <Abstract>Your Abstract.</Abstract>
        <OnlineResource xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="$(server_url)"/>
        <ContactInformation>
            <ContactPersonPrimary>
                <ContactPerson>Your Name</ContactPerson>
                <ContactOrganization>Your Organization</ContactOrganization>
            </ContactPersonPrimary>
            <ContactAddress>
                <AddressType>postal</AddressType>
                <Address>Your Street</Address>
                <City>Your City</City>
                <StateOrProvince></StateOrProvince>
                <PostCode>12345</PostCode>
                <Country>Germany</Country>
            </ContactAddress>
        </ContactInformation>
        <Fees>none</Fees>
        <AccessConstraints>This service is intended for research purposes only.</AccessConstraints>
    </Service>
```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

more variables and defaults for text informations about the service itselfs, fixes #45

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss (pull request #64)

more variables and defaults for text informations about the service itselfs, fixes #45
